MAUI/Maps/tooltip.md

```diff
@@ -273,9 +273,9 @@ The [`ShowMarkerTooltip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Ma
 
 Customize the tooltip appearance using the following properties.
 
-* **Background** - Change the background color of the tooltip in maps using the `MapTooltipSettings.Background` property.
-* **Padding** - Change the padding of the tooltip text in maps using the `MaptooltipSettings.Padding` property.
-* **TextStyle** - Change the tooltip text appearance such as `TextColor`, `FontSize`, `FontAttributes`, and `FontFamily` in maps using the `MapTooltipSettings.TextStyle` property.
+* **Background** - Change the background color of the tooltip in maps using the [`MapTooltipSettings.Background`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTooltipSettings.html#Syncfusion_Maui_Maps_MapTooltipSettings_Background) property.
+* **Padding** - Change the padding of the tooltip text in maps using the [`MaptooltipSettings.Padding`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTooltipSettings.html#Syncfusion_Maui_Maps_MapTooltipSettings_Padding) property.
+* **TextStyle** - Change the tooltip text appearance such as `TextColor`, `FontSize`, `FontAttributes`, and `FontFamily` in maps using the [`MapTooltipSettings.TextStyle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTooltipSettings.html#Syncfusion_Maui_Maps_MapTooltipSettings_TextStyle) property.
```

Comment:
API link not provided for

`TextColor`, `FontSize`, `FontAttributes`, and `FontFamily`

Reply:
I have provided the API link for it.